### ISSUE TITLE
Set default colormap in recipe settings

### ISSUE THREAD

**Issue by MirjamSchr** [tillbiskup/aspecd#1]
It would be great if you could set the default colormap already in the main settings of the recipe.

I imagine something like this: 
```
format:
  type: ASpecD recipe
  version: '0.2'
settings:
  default_package: aspecd
  default_colormap: turbo
datasets:
  - [...]
```

**Comment by tillbiskup** [comment]
The latest development version (0.9.0.dev18) has the "default_colormap" setting added to the recipe and should work...

**Comment by tillbiskup** [comment]
The feature has been implemented in the latest release 0.8.2, including not overriding the explicitly set colormap of a task by the default colormap.
